docs/api/query.md

```diff
@@ -124,6 +123,21 @@ for i, archetype in world:query(Position, Velocity).archetypes() do
 end
 ```
 
+```ts [typescript]

```

Comment:
This code group is redundant, you cannot get reliable types when inlining it. You should never do it from typescript.